tasks/section5.yml

```diff
@@ -551,15 +723,15 @@
       - rule_5.3.4
 
 - name: "SCORED | 5.3.4 | PATCH | Ensure password hashing algorithm is SHA-512"
-  command: /bin/true
+  command: authconfig --passalgo=sha512 --update
   tags:
       - level1
       - level2
       - patch
       - rule_5.3.4
 
 - name: "SCORED | 5.4.1.1 | AUDIT | Ensure password expiration is 90 days or less"
-  command: /bin/true
+  command: grep PASS_MAX_DAYS /etc/login.defs
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*